wremnants/datasets/dataset_tools.py

```diff
@@ -140,7 +140,7 @@ def is_zombie(file_path):
         return True
 
 def getDatasets(maxFiles=-1, filt=None, excl=None, mode=None, base_path=None, nanoVersion="v9", 
-                data_tag="TrackFitV722_NanoProdv2", mc_tag="TrackFitV718_NanoProdv1", oneMCfileEveryN=None, checkFileForZombie=False):
+                data_tag="TrackFitV722_NanoProdv2", mc_tag="TrackFitV718_NanoProdv1", oneMCfileEveryN=None, checkFileForZombie=False, useNewMCx4=False):
```

Comment:
this is left over from the other PR?

Reply:
Yes, I missed it. It was also doing nothing but I'll remove the argument. 
Before this PR is merged, we should ensure that all groups have copied the samples in their machines (otherwise as a temporary workaround to avoid crashes it should be sufficient to use options to exclude the new samples).